### PR TITLE
Fixed error that would not allow a deletion in a splice variant spanning 2 introns.

### PR DESCRIPTION
**Pull request by jairav**
Updated test files (xml and vcf) as well.

### CODE REVIEW

File: src/utils.py [hunk 1]
```diff
@@ -102,10 +102,10 @@ def parse_splice(cdsEffect, position, strand, fasta):
             mylen = abs(int(sub[0]) - int(sub[1]))+1
         else:
             if sub[0] != '0' and sub[1] != '0':
-                if int(main[1]) - int(main[0]) == 1:
-                    mylen = seqlen
-                else:
-                    raise ValueError('ERROR: main position has to match for both intronic else one of them must be an exon: ')
+                #if int(main[1]) - int(main[0]) == 1:
```
Comment (murrayln) [inline src/utils.py]: Why not just delete the commented out lines... they would be retained in git history for the file if we needed to revert to an earlier commit

File: test/data/expected.vcf [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ##fileformat=VCFv4.2
 ##source=foundation-xml-fhir
-##reference=file://genome.fasta
```
Comment (murrayln) [inline test/data/expected.vcf]: I guess the original file was created with the pytests

File: src/utils.py (hunk 1 above)
Comment (jairav) [reply src/utils.py]: done https://github.com/lifeomic/foundation-xml-fhir/commit/c2972b91f4a391bbb367710887d8bba1b8d1c6c0

**Comment by RyanRatcliff** [comment]
The test results file was updated but the source in the code to generate them was not, so the tests are failing.

**Comment by rcdilorenzo** [comment]
After a discussion with @jairav and @murrayln, here's the game plan. (Obviously, we're starting from a very small % test coverage and not a single e2e one.)

1. Remove the dead artifacts that aren't being used (e.g. `expected.json` and `sample.xml`)
2. Start a new branch with these artifacts and add a Jira issue for doing this e2e test properly.
3. Implement the card by using a small reference genome, crafted variants (including this rare one), and a full test from XML to VCF (2 days of work). This will give us the least amount of friction for future changes.

**Comment by jairav** [comment]
> After a discussion with @jairav and @murrayln, here's the game plan. (Obviously, we're starting from a very small % test coverage and not a single e2e one.)
> 
> 1. Remove the dead artifacts that aren't being used (e.g. `expected.json` and `sample.xml`)

This should only be `expected.json` as `sampled.xml` existed previously and was just reverted here : https://github.com/lifeomic/foundation-xml-fhir/pull/21/commits/6c0177d213c54c9fc60eb759717ca74665f965bc

> 2. Start a new branch with these artifacts and add a Jira issue for doing this e2e test properly.

Done : https://github.com/lifeomic/foundation-xml-fhir/commit/fea6e7624854dc1692977216f7c0d66e0ddddc42

> 3. Implement the card by using a small reference genome, crafted variants (including this rare one), and a full test from XML to VCF (2 days of work). This will give us the least amount of friction for future changes.

Jira Card for reference : https://lifeomic.atlassian.net/browse/SCI-21

**Review by rcdilorenzo — APPROVED** [review]
Looks good. Thanks for hashing out the discussion, everyone. 👍

**Review by RyanRatcliff — APPROVED** [review]

**Pull request merged**